CONTRIBUTING.rst

```diff
@@ -203,16 +203,67 @@ Docstrings
 
 PyVista uses Python docstrings to create reference documentation for our Python
 APIs. Docstrings are read by developers, interactive Python users, and readers
-of our online documentation. This page describes how to write these docstrings
+of our online documentation. This section describes how to write these docstrings
 for PyVista.
 
-* PyVista follows the ``numpydoc`` style for its docstrings. Please follow the
-  `numpydoc Style Guide`_.
+PyVista follows the ``numpydoc`` style for its docstrings. Please follow the
+`numpydoc Style Guide`_ in all ways except for the following:
+
 * Be sure to describe all ``Parameters`` and ``Returns`` for all public
   methods.
 * We strongly encourage you to add an example section. PyVista is a visual
   library, so adding examples that show a plot will really help users figure
   out what individual methods do.
+* With optional parameters, use ``default: <value>`` instead of ``optional``
+  when the parameter has a default value instead of ``None``.
+
+Sample docstring follows:
+
+.. code:: python
+
+    def slice_x(self, x=None, generate_triangles=False):
+        """Create an orthogonal slice through the dataset in the x-direction.
+
+        Yields a MutliBlock dataset of the three slices.
```

Comment:
```suggestion
        Yields a ``MultiBlock`` dataset of the three slices.
```
Or maybe even showcase intersphinx?
```suggestion
        Yields a :class:`MultiBlock <pyvista.MultiBlock>` dataset of the three slices.
```
(assuming this is within the usual length.)

Reply:
I think this whole line can be deleted.  This docstring stub is presumably referencing a slice operation in the x-normal direction, i.e. it only yields a single PolyData slice and could be described more in the `Returns` section if needed.